WeakAurasOptions/WeakAurasOptions.toc

```diff
@@ -90,8 +90,8 @@ AceGUI-Widgets\AceGUIWidget-WeakAurasSortedDropDown.lua
 AceGUI-Widgets\AceGUIWidget-WeakAurasToolbarButton.lua
 AceGUI-Widgets\AceGUIWidget-WeakAurasTwoColumnDropDown.lua
 AceGUI-Widgets\AceGUIWidget-WeakAurasSnippetButton.lua
+AceGUI-Widgets\AceGUIWidget-WeakAurasAnchorButtons.lua
 AceGUI-Widgets\AceGUIContainer-WeakAurasTreeGroup.lua
-AceGUI-Widgets\AceGUIWidget-WeakAurasSnippetButton.lua
```

Comment:
This appears to be accidental.

Reply:
The change doesn't belong to this PR, this lua file was added twice in the .toc (2 lines before)